resources/js/chat/conversation-view.tsx

```diff
@@ -66,13 +68,15 @@ export default class ConversationView extends React.Component<Props> {
       }
 
       // check whether the day-change header needs to be shown
-      if (isEmpty(conversationStack) || moment(message.timestamp).date() !== currentDay /* TODO: make check less dodgy */) {
+      if (isEmpty(conversationStack) || (moment(message.timestamp).date() !== currentDay && moment(message.timestamp).month() !== currentMonth && moment(message.timestamp).year() !== currentYear)) {
```

Comment:
Surely this cannot be the best way to do this?

Seems like something along the lines of `moment().isSame(message.timestamp, 'day')` may work better ([see docs](https://momentjs.com/docs/#/query/is-same/))?

Reply:
As `message.timestamp` is generated, using `moment().toISOString()`, it works.

Thanks.

I'm going to do that.